python-checks/src/main/java/org/sonar/python/checks/UnusedLocalVariableCheck.java

```diff
@@ -135,13 +139,37 @@ public PreciseIssue createIssue(SubscriptionContext ctx, Symbol symbol, Usage us
         issue.addQuickFix(quickFix);
       }
       return issue;
+    } else if (usage.kind().equals(Usage.Kind.ASSIGNMENT_LHS)) {
+      PreciseIssue issue = ctx.addIssue(usage.tree(), String.format(MESSAGE, symbol.name()));
+
+      Statement assignmentStatement = ((Statement) TreeUtils.firstAncestorOfKind(usage.tree(), Kind.ASSIGNMENT_STMT, Kind.ANNOTATED_ASSIGNMENT));
+
+      // This can only trigger if there is 1 assigned variable in the same statement, the other cases are handled by isSequenceUnpacking
+
```

Comment:
_Nitpick:_ I don't really think we need those white lines here. I don't really mind them though, no strong opinion.